irksome/stepper.py

```diff
@@ -161,6 +162,17 @@ def TimeStepper(F, butcher_tableau, t, dt, u0, **kwargs):
             it_solver_parameters, prop_solver_parameters,
             splitting, appctx, nullspace,
             num_its_initial, num_its_per_step)
+    elif stage_type == "dirkimex":
+        Fexp = kwargs.get("Fexp")
+        assert Fexp is not None
```

Comment:
Or, issue a warning that we don't have an explicit part and are defaulting to an IMEX method.  At ay rate, the failure/warning
 should have an error message to this effect.

Reply:
This was copied from [here](https://github.com/firedrakeproject/Irksome/blob/cbf6af426277fd84369c3fa6cdea211305d004be/irksome/stepper.py#L149), but I agree that a warning is a good idea.  Will add that now in both places.